### PR TITLE
View comments FE

### PR DESCRIPTION
**Pull request by dayanamdr**

### CODE REVIEW

File: codewell-fe/src/Comment/View.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+function ViewComments(props) {
+  const [viewComment, setViewComment] = useState([]);
+
+  const pageStyle = {alignItems: 'center'};
+  const containerStyle = {width: '80%'};
+  const cardStyle = {width: '100%'};
+
+  useEffect(() => {
+    fetch('/api/comments/' + props.id)
+    .then(response => response.json())
```
Comment (S7012MY) [inline codewell-fe/src/Comment/View.js:15]: indentation

File: codewell-fe/src/Comment/View.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        }
+      })
+  }, []);
+
+  return (
+    <Layout style={pageStyle}>
+      <Layout style={containerStyle}>
+        <List
+          itemLayout="horizontal"
+          dataSource={viewComment}
```
Comment (S7012MY) [inline codewell-fe/src/Comment/View.js:28]: viewComment -> comments

File: codewell-fe/src/Comment/View.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+          renderItem={item => (
```
Comment (S7012MY) [inline codewell-fe/src/Comment/View.js:29]: item -> comment?

**Review by S7012MY — CHANGES_REQUESTED** [review]

File: codewell-fe/src/Code/View.js [hunk 4]
```diff
@@ -59,6 +60,7 @@ function ViewCode() {
           </tbody>
         </table>
       </Layout>
+      <ViewComments id={id}/>
```
Comment (S7012MY) [inline codewell-fe/src/Code/View.js:63]: id->codeId

**Review by S7012MY — APPROVED** [review]

**Review by andreisamoila74 — APPROVED** [review]

**Pull request merged**